File: app/src/main/java/com/bth/running/statistics/StatisticsManager.java
package com.bth.running.statistics;

import com.bth.running.running.Run;

/**
 * @author Martin Macheiner
 *         Date: 06.10.2017.
 */

public interface StatisticsManager {

    Statistics getStatistics();

    void resetStatistics();

    void updateStatistics(Run run);

}

File: app/src/main/java/at/shockbytes/corey/running/statistics/StatisticsManager.kt
package at.shockbytes.corey.running.statistics


import at.shockbytes.corey.running.running.Run

/**
 * @author  Martin Macheiner
 * Date:    06.10.2017.
 */

interface StatisticsManager {

    val statistics: Statistics

    fun resetStatistics()

    fun updateStatistics(run: Run)

}
